xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.h

```diff
@@ -222,6 +222,7 @@ class CRenderManager
 
   struct SPresent
   {
+    SPresent();
```

Comment:
@FernetMenta looks like a race condition particularly involving the presentmethod field. We set the value from one thread here:

https://github.com/xbmc/xbmc/blob/3a27e3ba321dacdf7af05a3bab6bc39de9635738/xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp#L847

And we read it from a *different* thread here:

https://github.com/xbmc/xbmc/blob/3a27e3ba321dacdf7af05a3bab6bc39de9635738/xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp#L909

The order in which the two are executed can differ. I've observed both the correct & incorrect order.